html/elements/link.json

```diff
@@ -1102,7 +1105,8 @@
                   "version_added": "5.0"
```

Comment:
```suggestion
                  "version_added": "5.0",
                  "notes": "<code>as=\"document\"</code> is unsupported. See <a href='https://crbug.com/593267'>bug 593267</a>."
```

Reply:
Oh, thanks, I forgot that samsung internet is also chromium based!